modules/virt-removing-vm-delete-protection.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-removing-vm-delete-protection_{context}"]
+
+= Removing the virtual machine delete protection option
+
+When you enable delete protection on a VM (virtual machine), you ensure that the VM cannot be inadvertently deleted. You can also choose to disable the protection for a VM.
+
+As a cluster administrator, you can choose not to make the VM delete protection option available. VMs with delete protection already enabled retain that setting; for any new VMs that are created, enabling the option is not allowed.
+
+You configure making the option unavailable by using two YAML files, one for establishing a validation admission policy for the cluster and the other for creating the binding necessary to use the policy in the cluster.
```

Comment:
I don't think calling out that it is two YAML files is important. I'd focus on the actions. Maybe:

```suggestion
You can remove the delete protection option by establishing a validation admission policy for the cluster and then creating the necessary binding to use the policy in the cluster.
```